testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -223,7 +223,16 @@
 # cleanup steps
 When(/^I cleanup after Cobbler buildiso$/) do
   result, code = $server.run('rm -Rf /var/cache/cobbler')
-  raise "error during Cobbler buildiso cleanup.\nLogs:\n#{result}" if code.nonzero?
+  raise "Error during Cobbler buildiso cleanup.\nLogs:\n#{result}" if code.nonzero?
+end
+
+When(/^I cleanup Cobbler files$/) do
```

Comment:
```suggestion
When(/^I cleanup Cobbler files and restart apache and cobbler$/) do
```
or something that sounds better but I think it should be obvious that it's not just files being removed in this step